pyam/utils.py

```diff
@@ -32,6 +32,8 @@
                           + ['{}{}'.format(i, j) for i, j in itertools.product(
                               string.ascii_uppercase, string.ascii_uppercase)]))
 
+KNOWN_FUNCS = {'min': np.min, 'max': np.max, 'avg': np.mean, 'sum': np.sum}
```

Comment:
@peterkolp does it still missing `w.avg` key? or I misunderstand how to call `aggregate_region` method